### PR TITLE
Improve View meta-model with features that are already supported in our diagrams

### PR DESCRIPTION
**Pull request by gcoutable**
### Type of this PR 

- [ ] Bug fix
- [x] New feature or improvement
- [ ] Documentation
- [ ] Cleanup
- [ ] Test
- [ ] Build/Releng

### Issue(s)

https://github.com/eclipse-sirius/sirius-components/issues/598
https://github.com/eclipse-sirius/sirius-components/issues/599
https://github.com/eclipse-sirius/sirius-components/issues/600

### What does this PR do?

This PR adds the following features to the View meta-models:
- node border size
- edge width
- font formats for node label
- node label color

### CODE REVIEW

File: backend/sirius-web-view/src/main/resources/model/view.ecore [hunk 1]
```diff
@@ -73,6 +73,18 @@
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="borderRadius" lowerBound="1"
         eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EInt" defaultValueLiteral="3"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="shape" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="borderSize" lowerBound="1"
```
[the remark this answers is not in the document]
Comment (pcdavid) [reply backend/sirius-web-view/src/main/resources/model/view.ecore:76]: Actually, I thought the core of Sirius Web already supported dynamic border sizes and it was just a matter of passing the expression, but I was wrong. We can not do better in the View DSL itself, the core would need to change to support that and it's out of scope of this PR.

File: backend/sirius-web-view/src/main/resources/model/view.ecore [hunk 2]
```diff
@@ -81,6 +93,8 @@
         eType="#//ArrowStyle" defaultValueLiteral="None"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="targetArrowStyle" lowerBound="1"
         eType="#//ArrowStyle" defaultValueLiteral="OutputArrow"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="edgeWidth" lowerBound="1"
```
[the remark this answers is not in the document]
Comment (pcdavid) [reply backend/sirius-web-view/src/main/resources/model/view.ecore:96]: Same remark as above for my other comment.

**Review by pcdavid — APPROVED** [review]

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
Just to wait for the cool down, I'll have to review the impact of the separation of the business layer first.

**Review by sbegaudeau — APPROVED** [review]
(waiting for the build to merge)

**Pull request merged**